frontend/src/components/static_pages/team.css

```diff
@@ -3,8 +3,15 @@
   font-size: 1.2rem;
 }
 
-.meetTheTeam h2 {
-  margin-bottom: 20px;
+.heading {
+  font-weight: bold;
+  font-size: 36pt;
```

Comment:
i know she gave you these in pt but can we use pixels -> should be 48 

also this class is used on the Home Page -> lower mid but i think we want to make the change there too

Reply:
Michele specifically asked for all of the styling to be the same so I assumed that we should use the same class name, is that a bad practice that should be changed?

Reply:
Yeah thats why I think its fine. Just not sure if they realized the implications 👍 

Def best practice to have constant styles from page to page -> I am a big fan of the SMACSS Framework http://smacss.com/